lib/client/profile.go

```diff
@@ -444,7 +444,7 @@ func (p *ProfileStatus) KeyPath() string {
 // DatabaseCertPathForCluster returns path to the specified database access
 // certificate for this profile, for the specified cluster.
 //
-// It's kept in <profile-dir>/keys/<proxy>/<user>-db/<cluster>/<name>-x509.pem
+// It's kept in <profile-dir>/keys/<proxy>/<user>-db/<cluster>/<name>.crt
```

Comment:
Older clients will still read and write to that path, new clients will read/write the new path, and since they don't overlap at all, after an upgrade/downgrade it will just look like you aren't logged into any DBs and a fresh login should be triggered. We're not really doing anything for back compat here so I don't think we need to mention the path that old versions will use